luaui/Widgets_BAR/cmd_select_commander.lua

```diff
@@ -0,0 +1,32 @@
+function widget:GetInfo()
+  return {
+    name = "SelectCommander",
+    desc = "Press ctrl+c to select the commander",
+    author = "Teifion",
+    date = "26/10/2020",
+    license = "GNU GPL, v2 or later",
+    layer = 0,
+    enabled = true
+  }
+end
+
+function widget:Update()
```

Comment:
If you put the code in `Update` it gets called over and over. It should probably be based on some sort of event. I am unsure how BAR wires things up though.

Reply:
I've not seen documentation and the other widgets I looked at used update so I assumed it was called when the key was pressed. Happy to swap it out for a different function though.

Reply:
update is called every drawframe, so instead in widget:initialize, do:
widgetHandler:AddAction("select_commander", [function], [params], "t")

Have you managed to get this widget to run?

Reply:
I've not managed to get anything to run in development for this project, only the chobby one.

Reply:
User widgets are loaded from Beyond-All-Reason\data\LuaUI\Widgets_BAR (if the folders dont exist, create them).
Launch skirmish game vs nullai with chobby, press f11 to show widget list, click on widget to refresh, look in infolog.txt (search system if you have to for this file, it will be important for devving ) for any issues regarding loading/running your widget.

Reply:
User widgets are usually at the end of f11 list.